openedx/core/djangoapps/course_roles/docs/decisions/0001-course_level_roles.rst

```diff
@@ -0,0 +1,125 @@
+1. Course Level Roles
+######################
+
+
+Status
+******
+
+**Provisional** *2023-09-13*
+
+The status will be updated to *Accepted* upon completion of reimplementation.
+
+Context
+*******
+
+There is currently no single functionality being used by the majority of the codebases that are a part of the Open edX ecosystem
+that allows for adding roles (permission sets) that grant course related access for a user and can:
+- be assigned at the course level 
+- be assigned as an org-wide role (granting access to all courses within the org)
+- be assigned as an instance-wide role (granting access to all courses that are within the instance)
+- provide the flexibility to allow users to edit or create new roles (permission sets) that can be assigned to users
```

Comment:
This is not rendering correctly:
![image](https://github.com/openedx/edx-platform/assets/64440265/3a0c27bf-9ba1-4cfa-b265-d54649728825)

Reply:
The rendering issues include the tables below starting from line 39